### ISSUE TITLE
conda metapackages for installing sktime with soft dependencies

### ISSUE THREAD

**Issue by fkiraly** [alan-turing-institute/sktime#1142]
`pip` users can install `sktime` with soft dependencies via `pip install sktime[all_extras]`.

The same is not possible via `conda` currently, but could be achieved using metapackages, for instance.

Having that would be nice.

**Comment by aiwalter** [comment]
true, especially it could help some users to get `fbprophet` and `pystan` installed. At least more likely :)

**Comment by fkiraly** [comment]
indeed, I still have my `pip`/`conda` Frankenstein monster environment to have everything work properly...

**Comment by aiwalter** [comment]
haha same here. And I always hope nothing is breaking when I install something additionally

**Comment by freddyaboulton** [comment]
@fkiraly I took a crack at this here: https://github.com/conda-forge/sktime-feedstock/pull/11 . 

However, it's impossible to install all dependencies via conda:
- `pmdarima` causes version conflicts, you can see the conda output in the failed ci runs of the first commits.
- Even if `pmdarima` did not cause conflicts, it's not available on windows
- tbats and catch-22 do not have a package on conda forge

I've removed these from the recipe. I think that's a good first step. Follow-up steps could include filing issues for tbats and catch-22 to create conda packages and investigating the problems with pmdarima.

**Comment by fkiraly** [comment]
nice, thanks @freddyaboulton!

Do you have any suggestion on how to deal with the missing packages?
I think catch-22 we also wanted to move out of core mid-term.

So, should we close the issue or change the scope?

**Comment by freddyaboulton** [comment]
It was my pleasure @fkiraly ! 

I think we can close this issue and file smaller follow-up issues. 

I filed an issue for tbats to create a conda package here: https://github.com/intive-DataScience/tbats/issues/29 . Once that's done, I don't expect any problems adding tbats to the conda recipe.

If catch-22 will be removed sometime in the future, I think that just leaves pmdarima. We would need to figure out what's causing the version conflicts during the build and that could take some time. We can also file an issue for catch-22 to create a conda package but I leave it up to your judgment since you have a better understanding of the timeline for deprecating its use. 

So in short:

1. File issue for adding tbats to the recipe once they create a package. 
2. File an issue for adding pmdarima to the conda recipe.  
3. File an issue for updating the docs to let users know they can install (almost) all soft dependencies with `conda install -c conda-forge sktime-all-extras`

**Comment by fkiraly** [comment]
Closed with thanks. New issue is #1227.
I already solved the docs item in #1226.
